business-central-parent/business-central-home-page-product/src/main/java/org/kie/bc/client/home/AbstractHomeProducer.java

```diff
@@ -56,18 +58,31 @@ public AbstractHomeProducer(final PlaceManager placeManager,
         this.shortcutHelper = shortcutHelper;
     }
 
-    public HomeModel get() {
+    public HomeModel get(ProfilePreferences profilePreferences) {
+        this.profilePreferences = profilePreferences;
         final HomeModel model = new HomeModel(translationService.format(Constants.Heading),
                                               translationService.format(Constants.SubHeading),
-                                              "images/community_home_bg.jpg");
+                                              "images/product_home_bg.png");
+
+        if(profilePreferences.isRulesAndPlanner()) {
```

Comment:
Formatting issue (missing space after if) multiple times in this PR (ctrl+f  "if(" )